cms/static/js/hera/components/EndSurvey.jsx

```diff
@@ -1,144 +1,267 @@
 
                             <div className="end-survey__row-buttons">
-                                <button className="end-survey__row-btn is-remove" type="button">
-                                    <i className="fa fa-trash-o" aria-hidden="true" />
-                                    Remove row
-                                </button>
-                                <button className="end-survey__row-btn is-add" type="button">
-                                    <i className="fa fa-plus-circle" aria-hidden="true" />
-                                    Add row with items
+                                <button className="end-survey__row-btn is-add" type="button" onClick={this.addEndSurveyQuestion}>
+                                    <i className="fa fa-plus-circle" aria-hidden="true"/>
+                                    Add a question
                                 </button>
                             </div>
                         </div>
+                        <div className="end-survey__row">
+                            <SurveyQuestion
+                                index={1}
+                                isConfidence={true}
+                                changeHandler={this.endSurveyConfidenceChanged}
+                                {...confidence}
+                                />
+                        </div>
                     </div>
                     <div className="author-block__image is-small">
-                        <div className="author-block__image-selector">
-                            <i className="fa fa-picture-o" aria-hidden="true" />
-                            <br/>
-                            <button type="button" className="author-block__image-selector__btn">+ Add image</button>
-                        </div>
+                        {
+                            imgUrl ? (
+                                <img className="end-survey__img" src={imgUrl} alt=""/>
+                            ) : (
+                                <div className="author-block__image-selector">
+                                    <i className="fa fa-picture-o" aria-hidden="true"/>
+                                </div>
+                            )
+                        }
                     </div>
                 </div>
-                <div className="author-toolbar is-right">
-                    <div className="author-toolbar__row is-small">
+                <div className="author-toolbar is-end">
+                    <div className="author-toolbar__row">
                         <div className="author-toolbar__row-holder">
                             <input
                                 className="author-toolbar__field"
                                 type="text"
                                 placeholder='Paste URL of the image'
+                                value={imgUrl}
+                                onChange={this.changeImgUrl.bind(this)}
```

Comment:
Same question as above: why not to do the binding in the constructor?

Reply:
done